### ISSUE TITLE
Enabling `linux_aarch64` builds

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/dgl-feedstock#13]
Would be great to have `linux_aarch64` packages of `dgl` as well

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render

**Comment by mikemhenry** [comment]
Good idea! Is there a way to do build native `osx-arm64` yet? I am not sure if `dgl` is set up for cross compilation...

**Comment by jakirkham** [comment]
Ah was thinking about Linux

Added to the migrator for macOS ARM ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4821 )

Unfortunately no native macOS ARM builds yet

We can look at cross-compilation changes, but we may still be missing dependencies (`metis` and `nanoflann` plus possibly `pytorch` depending)
